pkg/proc/variables.go

```diff
@@ -625,18 +625,18 @@ func (scope *EvalScope) PackageVariables(cfg LoadConfig) ([]*Variable, error) {
 	var vars []*Variable
 	reader := scope.DwarfReader()
 
-	var utypoff dwarf.Offset
+	var utypeoff dwarf.Offset
```

Comment:
I'm pretty sure this is a reasonable change

Reply:
There's a lot of places where we use typ instead of type, I would leave those as they are for consistency.